pages/common/docker-build.md

```diff
@@ -0,0 +1,28 @@
+# docker build
+
+> Build an image from a Dockerfile.
+> More information: <https://docs.docker.com/engine/reference/commandline/build/>.
+
+- Build a docker image using the Dockerfile in the current directory:
+
+`docker build .`
+
+- Build a docker image from a Dockerfile at a specified URL:
+
+`docker build {{github.com/creack/docker-firefox}}`
+
+- Build a docker image and tag it:
+
+`docker build --tag {{name:tag}} .`
+
+- Do not use the cache when building the image:
+
+`docker build --no-cache --tag {{name:tag}} .`
+
+- Build a docker image using a specific Dockerfile:
+
+`docker build -f {{Dockerfile}} .`
```

Comment:
Perhaps you missed my comment? Looking at the docs, it would appear there is indeed a longer form of the `-f` argument:
```suggestion
`docker build --file {{Dockerfile}} .`
```